src/plugins/intel_gpu/tests/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -2169,9 +2170,18 @@ struct dynamic_fully_connected_gpu : ::testing::TestWithParam<fully_connected_dy
                                                                               input_data_vec,
                                                                               weights_data_vec,
                                                                               bias_data_vec);
-            for (int b = 0; b < batch_size; b++) {
-                for (int ofm = 0; ofm < output_f; ofm++) {
-                    ASSERT_EQ(ref_result[b * output_f + ofm], output_ptr[b * output_f + ofm]);
+
+            if (engine.get_device_info().supports_immad) {
+                for (int b = 0; b < batch_size; b++) {
+                    for (int ofm = 0; ofm < output_f; ofm++) {
+                        EXPECT_NEAR(ref_result[b * output_f + ofm], output_ptr[b * output_f + ofm], 0.5);
```

Comment:
0.5 seems quite large. What is the output data range? What about using `get_default_tolerance` function to set different threshold based on data type?

Reply:
Ok I will apply this fix.
If data_types are s8, s8, fp32 for input0, weight and output, then accuracy type is s32.
I think this makes main difference. So for now, I will use default_tolerance with input_type.

Reply:
Fixed